### DISCUSSION TITLE
Is there a way to use the Data Importer (FIDI) with multiple user accounts?

### DISCUSSION THREAD

**Discussion by nabeelmoeen** [firefly-iii/firefly-iii#7827] [Firefly III]
hi,

I believe Firefly iii core application allows me to create multiple user profiles so each user can independently use Firefly on the same installation.
I wanted to confirm if the Data Importer has been updated to support multi user profiles as well? when searching I came across the issue: https://github.com/firefly-iii/firefly-iii/issues/3722 which is now marked as closed/completed.
But was unable to locate documentation on whether it was actually integrated in the new Data Importer.
I am using FIDI v1.3.3

**Comment by JC5** [comment]
https://docs.firefly-iii.org/data-importer/faq/general/#is-the-data-importer-multi-user

**Reply by nabeelmoeen** [reply]
Thank you very much for this

**Reply by nabeelmoeen** [reply]
sorry, follow up question. I am assuming this wont work with remote_user authentication, since that works with Personal access tokens only?
https://docs.firefly-iii.org/data-importer/installation/configuration/

**Reply by JC5** [reply]
Good point, I'm afraid not. I will add it to the docs. You could always set up multiple instances though.

**Reply by nabeelmoeen** [reply]
thanks for the prompt responses. 
and, yep, I will try and setup a separate importer instance for the second profile.
